frontend/src/features/admin-form/AdminFormCreatePage.stories.tsx

```diff
@@ -82,6 +93,17 @@ TabletAllFields.parameters = {
   chromatic: { viewports: [viewports.md] },
   msw: buildMswRoutes({ form_fields: MOCK_FORM_FIELDS_WITH_MYINFO }),
 }
+export const TabletLoading = Template.bind({})
+TabletLoading.parameters = {
+  viewport: {
+    defaultViewport: 'tablet',
+  },
+  chromatic: { viewports: [viewports.md] },
+  msw: buildMswRoutes(
+    { form_fields: MOCK_FORM_FIELDS_WITH_MYINFO },
+    'infinite',
+  ),
+}
```

Comment:
nit: same here. 

nit: should we also use the `getTabletViewParameters` storybook util function?

Reply:
sounds good, done for this + existing stories!